common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -26,7 +26,7 @@
             var match;
 
             this.$el = options.el;
-            this.readOnly = options.readOnly;
```

Comment:
An obvious question,
Is it because we are not passing `readOnly` in the init method here? Why we are changing this?